presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -61,8 +61,6 @@
 import com.google.common.collect.ImmutableSet;
```

Comment:
tests are in `testOrderByWithOutputColumnReference` methods. There are no unit tests for `AggregationAnalyzer`